shared/chat/conversation/list.desktop.js

```diff
@@ -248,7 +255,7 @@ class ConversationList extends Component<void, Props, State> {
 }
 
 class CellSizeCache extends defaultCellMeasurerCellSizeCache {
-  _indexToID: (index: number) => number;
+  _indexToID: (index: number) => ?number;
```

Comment:
@chrisnojima 
Without this change:
```
chat/conversation/list.desktop.js:45
 45:     this._cellCache = new CellSizeCache(this._indexToID)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ constructor call
 71:       return id
                  ^^ possibly uninitialized variable. This type is incompatible with
258:   _indexToID: (index: number) => number;
                                      ^^^^^^ number
```
The code is:
```js
      let id
      if (message) {
        if (message.type === 'Timestamp') {
          id = message.timestamp
        } else {
          // We want a stable key -- messages have an outboxID but no
          // messageID, then later gain a messageID.  So if we prefer
          // outboxIDs to messageIDs for the key, every row keeps its key.
          id = message && (message.outboxID || message.messageID)
        }
      }
      if (id == null) {
        console.warn('id is null for index:', messageIndex)
      }
      return id
    }
```
So the function's capable of returning null (even though we never want it to), but the flow type was requiring a number.